public/ide/views/home/home.controller.js

```diff
@@ -6,7 +6,65 @@
         .module("genXapp")
         .controller("HomeController", HomeController);
 
-    function HomeController() {
+    /* Modified by Anupma : Added Controller code for login and register*/
+    function HomeController($rootScope, $location, UserService) {
 
+        var vm = this;
+        vm.login = login;
+        vm.register = register;
+        vm.register = register;
+        vm.toggleNavBar = toggleNavBar;
+
+        function toggleNavBar(val) {
+            if (val == true) {
+                document.getElementById("mySidenav").style.width = "60%";
+                document.getElementById("mySidenav").style.display = "block";
+            }
```

Comment:
Anupma, please use ng-if or ng-show instead of changing styles from the controller.